mx.truffleruby/mx_truffleruby.py

```diff
@@ -119,9 +119,9 @@ def store_jvm_args(self):
             classpath.remove(jar)
 
         with open(self.jvm_args_file, 'w') as f:
-            f.write('bootclasspath=' + ':'.join(bootclasspath) + '\n')
-            f.write('classpath=' + ':'.join(classpath) + '\n')
-            f.write('properties=("' + '" "'.join(properties) + '")\n')
+            f.write('bootclasspath=\\\n' + ':\\\n'.join(bootclasspath) + '\n\n')
+            f.write('classpath=\\\n' + ':\\\n'.join(classpath) + '\n\n')
+            f.write('properties=(\n"' + '"\n"'.join(properties) + '"\n)\n')
```

Comment:
Can you paste what it looks like with this change?

Reply:
```sh
bootclasspath=\
$root_parent/graal/sdk/mxbuild/dists/graal-sdk.jar:\
$root_parent/graal/truffle/mxbuild/dists/truffle-api.jar:\
$root_parent/graal/sdk/mxbuild/dists/launcher-common.jar

classpath=\
$root_parent/graal/truffle/mxbuild/dists/truffle-nfi.jar:\
$root_parent/graal/tools/mxbuild/dists/truffle-profiler.jar:\
$root_parent/graal/tools/mxbuild/dists/chromeinspector.jar:\
$root/mxbuild/dists/truffleruby-launcher.jar:\
$root/mxbuild/dists/truffleruby.jar:\
$root_parent/sulong/build/sulong.jar

properties=(
"-Dtruffle.nfi.library=$root_parent/graal/truffle/mxbuild/darwin-amd64/truffle-nfi-native/bin/libtrufflenfi.dylib"
"-Dpolyglot.llvm.libraryPath=$root_parent/sulong/mxbuild/sulong-libs"
)
```

Reply:
Looks nice :)